### ISSUE TITLE
update sdk-template to latest version

### ISSUE THREAD

**Issue by ulex** [JetBrains/resharper-devguide#51]
Hi, @citizenmatt 

Could you please update sdk-template to latest version?
and maybe set `fetch.recurseSubmodules`? 

It has fix for this issue
https://github.com/JetBrains/sdkdocs-template/issues/2

**Comment by citizenmatt** [comment]
Yep, will do it later today. There are a couple of updates to `sdkdocs-template` that need to come across. Haven't seen `fetch.recurseSubmodules`, I'll check it out.
